src/main/java/uk/gov/hmcts/reform/blobrouter/services/report/ReportService.java

```diff
@@ -42,6 +52,17 @@ public ReportService(ReportRepository reportRepository) {
             .collect(toList());
     }
 
+    public List<EnvelopeCountSummaryReportItem> getCountFor(LocalDate date, boolean includeTestContainer) {
+        List<String> containersList = serviceConfiguration.getSourceContainers();
+        long start = System.currentTimeMillis();
+        final List<EnvelopeCountSummaryReportItem> reportResult = reportRepository.getReportFor(
+            date, containersList).stream()
+            .filter(it -> includeTestContainer || !Objects.equals(it.container, TEST_CONTAINER))
```

Comment:
if we want filtering we need to configure it per environment as we use these containers in lower environments. I think we do not need filtering.